From 477ee7649523b6bab8a0a07dc3e0226a1182797b Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Tue, 1 Jul 2025 14:05:59 +0200
Subject: [PATCH] dt-bindings: media: st: csi: correct dtbs_check warning

Shorten the binding title in order to correct dtbs_check warning.

Change-Id: I817ec1dbcd6ab3b834523e8437f8b27da258e0af
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/471687
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
---
 Documentation/devicetree/bindings/media/st,stm32-csi.yaml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/media/st,stm32-csi.yaml b/Documentation/devicetree/bindings/media/st,stm32-csi.yaml
index 326b430f85f5..4bc2daba2284 100644
--- a/Documentation/devicetree/bindings/media/st,stm32-csi.yaml
+++ b/Documentation/devicetree/bindings/media/st,stm32-csi.yaml
@@ -4,8 +4,7 @@
 $id: http://devicetree.org/schemas/media/st,stm32-csi.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: STMicroelectronics STM32 CSI controller allowing to connect a CSI
-       based camera to the DCMIPP camera pipeline.
+title: STMicroelectronics STM32 CSI controller
 
 maintainers:
   - Alain Volmat <alain.volmat@foss.st.com>
-- 
2.53.0

